repos/fdbt-site/src/utils/index.ts

```diff
@@ -261,3 +263,50 @@ export const objectKeyMatchesExportNameExactly = (objectKey: string, exportName:
 export const fareTypeIsAllowedToAddACap = (fareType: string): boolean => {
     return !!fareType && ['single', 'return', 'flatFare'].includes(fareType);
 };
+
+export const getMyUniqueServices = (
+    services: { id?: string | number; lineId: string; startDate: string; endDate?: string }[],
```

Comment:
If we're using id's to store which ones are unique, why is id nullable on the input?

Reply:
@NitishAroraKPMG needs a response please